src/mapgen.h

```diff
@@ -114,6 +114,7 @@ enum MapgenType {
 	MAPGEN_V5,
 	MAPGEN_V6,
 	MAPGEN_V7,
+	MAPGEN_CARPATHIAN,
```

Comment:
This enum was created in this form in hmmmm's mapgen reorganisation commit https://github.com/minetest/minetest/commit/92705306bfb4994107a43514f29997cea15d48dc
So i'm sure it's fine as it is. 
However any new mapgens should perhaps be added after 'invalid'?